app/services/onlinetesting/OnlineTestExpiryService.scala

```diff
@@ -42,16 +43,13 @@ class OnlineTestExpiryServiceImpl(
   newHeaderCarrier: => HeaderCarrier
 )(implicit executor: ExecutionContext) extends OnlineTestExpiryService {
 
-  private final val ExpiredStatus = "ONLINE_TEST_EXPIRED"
   private implicit def headerCarrier = newHeaderCarrier
 
   override def processNextExpiredTest(): Future[Unit] = {
```

Comment:
I think if the name of the method is "processNextExpiredTest", then this line:

otRepository.nextApplicationPendingExpiry

should be:
otRepository.nextApplicationExpiredTest

nextApplicationPendingExpiry is supposed to return an application (by the name), if it is an online tests what is returning, it should say: nextExpiredOnlineTest.